scripts/install_dependencies

```diff
@@ -2,20 +2,20 @@
 
 set -e
 
-CATALINA_HOME=/usr/share/tomcat7-codedeploy
+CATALINA_HOME=/usr/share/tomcat-codedeploy
 
 # Tar file name
-TOMCAT7_CORE_TAR_FILENAME='apache-tomcat-7.0.72.tar.gz'
-# Download URL for Tomcat7 core
-TOMCAT7_CORE_DOWNLOAD_URL="http://mirror.olnevhost.net/pub/apache/tomcat/tomcat-7/v7.0.72/bin/$TOMCAT7_CORE_TAR_FILENAME"
+TOMCAT_CORE_TAR_FILENAME='apache-tomcat-8.5.11.tar.gz'
+# Download URL for Tomcat core
+TOMCAT_CORE_DOWNLOAD_URL="http://apache.mirror.digitalpacific.com.au/tomcat/tomcat-8/v8.5.11/bin/$TOMCAT_CORE_TAR_FILENAME"
```

Comment:
Looks like the URL "http://apache.mirror.digitalpacific.com.au/tomcat/tomcat-8/v8.5.11/bin/apache-tomcat-8.5.11.tar.gz" is broken. Also lets stick to the ones listed in the official Apache mirrors.